tests/unit/test_gbq.py

```diff
@@ -100,6 +118,45 @@ def test_to_gbq_with_no_project_id_given_should_fail(monkeypatch):
     assert "Could not determine project ID" in str(exception)
 
 
+def test_to_gbq_read_projectid_from_service_account_credentials(
+    min_bq_version
+):
+    import pkg_resources
+
+    credentials, _ = mock_get_service_account_credentials()
+    pandas_version = pkg_resources.parse_version("0.24.0")
```

Comment:
Could you remove the version mocking in this and the other test? That was only present in the other tests to look for a specific warning message related to that.

Reply:
Done in 9035771667204c946d74570ad2498567a0f778f4.